wallet/res/layout/activity_accept_invite.xml

```diff
@@ -3,6 +3,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/white"
```

Comment:
White background will stay white in the dark mode. Unless this is intended, better to use `secondaryBackground`.

Reply:
This is to replace the transparent background with the correct background color.

Reply:
Fixed